include/bluetooth/conn.h

```diff
@@ -413,8 +446,10 @@ struct bt_conn_cb {
 	 *
 	 *  @param conn Connection object.
 	 *  @param level New security level of the connection.
+	 *  @param reason Security error. Zero for success, non-zero otherwise.
```

Comment:
You have "reason" here, but in the definition you use "err"

Reply:
Thanks. fixed.